File: 02/agentApp/agentApp/ViewController.m
//
//  ViewController.m
//  agentApp
//
//  Created by Taylor Frost on 5/24/16.
//  Copyright © 2016 FrostEnterprises. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
{
    NSString *uname;
}
@property (weak, nonatomic) IBOutlet UITextField *usernameTextField;
@property (weak, nonatomic) IBOutlet UIButton *authenticateButton;
@property (weak, nonatomic) IBOutlet UITextField *passwordTextField;
@property (weak, nonatomic) IBOutlet UILabel *greetingLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    
    uname = @"Frost";
    
    
    
}
- (IBAction)authenticateTapped:(UIButton *)sender {
    if ([self.usernameTextField.text isEqualToString:@"Taylor Frost"] && [self.passwordTextField.text
         isEqualToString:@"111111"]) {
        
            self.view.backgroundColor = [UIColor greenColor];
        
                self.greetingLabel.text = [NSString stringWithFormat:@"Good evening, Agent %@", uname];

        
        
        
    }   else {
        self.view.backgroundColor = [UIColor redColor];
    
    }
    
}
@end

File: 15/AgentAppSwift/ViewController.swift
//
//  ViewController.swift
//  AgentAppSwift
//
//  Created by Taylor Frost on 6/13/16.
//  Copyright © 2016 Taylor Frost. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UITextFieldDelegate {

    @IBOutlet weak var UsernameLabel: UITextField!
    
    @IBOutlet weak var PasswordLabel: UITextField!
    
    @IBOutlet weak var AuthenticateButton: UIButton!
    @IBOutlet weak var greetingLabel: UILabel!
    
 
    func textFieldShouldReturn(textField:UITextField) ->
        Bool {


            if textField == UsernameLabel {
                PasswordLabel.becomeFirstResponder()
            } else {
                textField.resignFirstResponder()
            }
            return true
}

    @IBAction func authTapped(sender: UIButton) {
    

    let secretAgent = "Frost"
    
    if (self.UsernameLabel.text! == "Taylor") &&
        (self.PasswordLabel.text! == "password") {
    
    self.view.backgroundColor = UIColor.greenColor()
    self.greetingLabel.text = "Good Evening, Agent \(secretAgent)"
    } else {
    
    self.view.backgroundColor = UIColor.redColor()
    }

}

}


